package.json

```diff
@@ -75,13 +75,16 @@
   "dependencies": {
     "axe-core": "2.2.1",
     "chrome-devtools-frontend": "1.0.422034",
+    "configstore": "^3.1.0",
```

Comment:
just a note so we remember, we should explore `optionalDependencies` for these (and the other shimmed ones) in the future to at least signal that these aren't necessary for folks looking for a slimmer bundle of dependencies.